src/Lightning.ts

```diff
@@ -374,6 +374,7 @@ export const LightningMixin = (superclass) =>
                 payerUser: this.user,
                 payeeUser,
                 memoPayer,
+                shareMemoWithPayee: pushPayment ? true : false,
```

Comment:
If `pushPayment` can only have boolean values then we don't need the ternary here.

Maybe also consider renaming `pushPayment` to something that indicates it being a boolean.

Reply:
I think `isPushPayment` should work

Reply:
Get rid of the ternary?

  `shareMemoWithPayee: isPushPayment,`

Reply:
> Get rid of the ternary?
> 
> `shareMemoWithPayee: isPushPayment,`

Yep that's what I did

Reply:
Fixed now